listenbrainz/db/tests/test_user.py

```diff
@@ -19,10 +19,10 @@ def test_update_token(self):
 
     def test_update_last_login(self):
         user = db_user.get_or_create('testlastloginuser')
-        val = int(time.time())
-        db_user.update_last_login(user['musicbrainz_id'], val)
+        db_user.update_last_login(user['musicbrainz_id'])
         user = db_user.get_by_mb_id(user['musicbrainz_id'])
-        self.assertEqual(val, int(user['last_login'].strftime('%s')))
+        # after update_last_login, the val should be equal to the current time
+        self.assertEqual(int(time.time()), int(user['last_login'].strftime('%s')))
```

Comment:
this is dangerous, because 99% of the time it might work, but then suddenly there may be a case where we do the above line just before the time increases by a second.
I would prefer to create a user with a last login time obviously in the past - you could do this by doing an sql query to get `now() - interval 1 day` or choose some time in june 2017 - and then do the update and check that this value is greater than the original value that you set it to